--- Androidorig.bp	2021-02-09 20:53:53.558074000 +0600
+++ Androidpached.bp	2021-02-10 14:55:59.869002000 +0600
@@ -21,7 +21,7 @@
     name: "generated_kernel_includes",
 
     // The headers make command
-    cmd: "$(PATH_OVERRIDE_SOONG) $(KERNEL_MAKE_CMD) $(KERNEL_MAKE_FLAGS) -C $(TARGET_KERNEL_HEADERS) O=$(KERNEL_BUILD_OUT_PREFIX)$(genDir) ARCH=$(KERNEL_ARCH) $(KERNEL_CROSS_COMPILE) headers_install",
+    cmd: "$(PATH_OVERRIDE_SOONG) $(KERNEL_MAKE_CMD) $(KERNEL_MAKE_FLAGS) -C $(TARGET_KERNEL_HEADER_SOURCE) O=$(KERNEL_BUILD_OUT_PREFIX)$(genDir) ARCH=$(KERNEL_ARCH) $(KERNEL_CROSS_COMPILE) headers_install",
 
     // Directories that can be imported by a cc_* module generated_headers property
     export_include_dirs: [
@@ -32,7 +32,7 @@
     ],
 
     // Sources for dependency tracking
-    dep_root: "$(TARGET_KERNEL_HEADERS)",
+    dep_root: "$(TARGET_KERNEL_HEADER_SOURCE)",
     dep_files: [
         "Makefile",
         "include/**/*",
