--- BoardConfigSoong.mk	2020-07-24 11:05:28.747781000 +0000
+++ BoardConfigSoong.mk	2020-07-21 13:06:16.596802000 +0000
@@ -9,8 +9,7 @@
     KERNEL_MAKE_FLAGS \
     PATH_OVERRIDE_SOONG \
     TARGET_KERNEL_CONFIG \
-    TARGET_KERNEL_SOURCE \
-    TARGET_KERNEL_HEADERS
+    TARGET_KERNEL_HEADER_SOURCE
 
 # Setup SOONG_CONFIG_* vars to export the vars listed above.
 # Documentation here:
